src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Components/Header.razor.cs

```diff
@@ -1,16 +1,18 @@
-﻿namespace TodoTemplate.App.Components;
+﻿using Bit.Tooling.SourceGenerators;
```

Comment:
instead of adding this using in each file, add it once in the `_Improts.razor` file.